frontend/src/tabs/zoom.pug

```diff
@@ -10,7 +10,14 @@
       v-bind:udate="info.untilDate",
       v-bind:avgsize="info.avgCommitSize")
 
-  .zoom__day(v-for="day in info.user.commits")
+  .zoom__week(v-if="filterTimeFrame === 'week'", v-for="week in info.user.commits")
+    template(v-if="week.insertions > 0")
+      h3 Week of {{ week.date }}
+      ul
```

Comment:
These three lines are quite similar with the below one, is it possible to write these two in one template? So you can also avoid the problem I mentioned before.